README.md

```diff
@@ -55,13 +57,33 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
+
+**Note:** Commands can be executed on Linux, Mac, and Windows (using Powershell)
+
+1. Open a terminal window and clone the forked copy of the docs repo by running:
+```sh
+$ git clone https://github.com/mattermost/docs.git
+```
+2. Install [pipenv](https://docs.pipenv.org/) by using:
+```sh
+$ brew install pipenv   (for Mac users, if you have brew installed)
```

Comment:
To ensure clarity, reference "brew" as "Homebrew".

In addition, consider moving the platform/operating system references from the end of the sentence to the beginning to make them more prominent/visible.

Reply:
Sure, updated.